Jenkinsfile

```diff
@@ -18,7 +18,6 @@ edgeXBuildGoParallel(
     project: 'edgex-go',
     goVersion: '1.13',
     dockerFileGlobPath: 'cmd/**/Dockerfile',
-    testScript: 'make test raml_verify',
     buildScript: 'make build',
     publishSwaggerDocs: true
 )
```

Comment:
CI default is api/openApi/... So also must add for the non-default paths.
```
swaggerApiFolders: ['openapi/v1', 'openapi/v2']
```

Reply:
@lenny-intel For this pipeline, the [default](https://github.com/edgexfoundry/edgex-global-pipelines/blob/master/vars/edgeXBuildGoParallel.groovy#L426) is already what you have listed. You do not have to override if you don't want to.

Reply:
@ernestojeda , note that we have dropped the leading `api` folder.
`api/openapi/v1` => `openapi/v1`

Reply:
Yes. @cherrycl opened a PR the edgex-global-pipelines to update the defaults

Reply:
@cherrycl Please add the overrides as @lenny-intel mentioned above. This will be the easiest way to get this PR merged.

```
swaggerApiFolders: ['openapi/v1', 'openapi/v2']
```

Reply:
> Yes. @cherrycl opened a PR the edgex-global-pipelines to update the defaults

Ahh, was not aware. Yea we'll have to go with the override for now and then swing back around and remove it if we feel it is worth removing. THX!

Reply:
@lenny-intel @ernestojeda Added 'make test' back and swaggerApiFolders.